docs/content/en/docs/Creating derivatives/build_disk.md

```diff
@@ -2,64 +2,67 @@
 title: "Build disk images with Elemental"
 linkTitle: "Build disk images with Elemental"
 weight: 4
-date: 2023-08-31
+date: 2024-08-20
 description: >
   This section documents the procedure to build disk images using elemental
 ---
 
-Requirements:
+In order to build a RAW disk image we rely on `elemental build-disk` command. It accepts a YAML file denoting the sources to bundle in the RAW image. In addition it can also overlay custom files or use container images from a registry as packages.
```

Comment:
Do we have a page explaining what we mean when we talk about "RAW disk image" ?
Why do we write "raw" in CAPS ?